src/components/transaction/AssignTasksWrapper/index.jsx

```diff
@@ -0,0 +1,282 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { ReactComponent as NotesIcon } from 'assets/icons/overview.svg';
+import { ReactComponent as BuyerAttorneyIcon } from 'assets/icons/transaction-buyer-attorney.svg';
+import { ReactComponent as CoordinatorIcon } from 'assets/icons/transaction-coordinator2.svg';
+import { ReactComponent as HomeInsuranceIcon } from 'assets/icons/transaction-home-insurance.svg';
+import { ReactComponent as TitleInsuranceIcon } from 'assets/icons/transaction-insurance.svg';
+import { ReactComponent as LenderIcon } from 'assets/icons/transaction-lender.svg';
+import { ReactComponent as SellerAttorneyIcon } from 'assets/icons/transaction-seller-attorney.svg';
+import Button from 'components/shared/Button';
+import DatePickerInput from 'components/shared/DatePickerInput';
+import DropdownMenu from 'components/shared/DropdownMenu';
+import TextAreaInput from 'components/shared/FormTextArea';
+import Table from 'components/shared/Table';
+import { TableCell, TableRow } from 'components/shared/Table/table-styles';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import assignTasksSchema from './assign-tasks-wrapper-schema';
+import {
+  AssignTasksForm,
+  ButtonsContainer,
+  InputsContainer,
+  PartiesInputsContainer,
+  ProcessText,
+  SectionContainer,
+} from './assign-tasks-wrapper.styles';
+
+const initialTableContent = [
+  {
+    assignToDefaultValue: null,
+    assignToName: 'initialContractName',
+    assignToOptions: ['John', 'Mark', 'Frank'],
+    dateDefaultValue: null,
+    dateName: 'initialContractDate',
+    id: 1,
+    process: 'Initial Contract',
```

Comment:
I don't think these are needed.

```suggestion
    assignToName: 'initialContractName',
    assignToOptions: ['John', 'Mark', 'Frank'],
    dateDefaultValue: null,
    dateName: 'initialContractDate',
    id: 1,
    process: 'Initial Contract',
```

Reply:
i need them in the future to enter the values that are returned from backend